### ISSUE TITLE
Cannot set icon for exe

### ISSUE THREAD

**Issue by Garrom** [godotengine/godot#27765]
**Godot version:**
GoDot 3.1 Stable/ rcedit 1.1.1/wine 3.0.2

**OS/device including version:**
Ubuntu 19.04

**Issue description:**
No matter what I try, every time I export for windows .exe file does not have icon. There is no error in log. I tried everything. Using rcedit in terminal gives me error not shown in godot :0009:fixme:ver:GetCurrentPackageId (0x33fe24 (nil)): stub .I'm giving up, I'll just leave here issue in case someone know what to do with this. Nobody knew on discord, nobody on reddit, I don't know why I'm still trying...

**Comment by groud** [comment]
Related to / duplicate of #21347

**Comment by Calinou** [comment]
@JacobTheHeartless Can you reproduce this with a more recent WINE version (4.17 as of writing)?

**Comment by hickop** [comment]
Tried with wine 5.4, still the same issue.

**Comment by akien-mga** [comment]
Please document:
- rcedit version used
- export settings used in the Windows preset

It sounds like invalid input is passed to rcedit, so it fails.

**Comment by hickop** [comment]
Figured my .ico file was missing icons at various sizes, it works with wine 5.4 and rcedit 1.1.1

**Comment by akien-mga** [comment]
Thanks. I suspect @JacobTheHeartless had a similar issue then.

We should look into whether we can do better error reporting when rcedit fails.

**Comment by Mickeon** [comment]
I will ask the same question as above because I feel like 5 years later so many things have changed on this side of the codebase. It also hardly is a "good first issue".

**Comment by ghildim** [comment]
Anyone still looking into this issue?

**Comment by naseemagithub** [comment]
“Hi! I’m a beginner and would love to work on this. Can I take this up?”

**Comment by Calinou** [comment]
rcedit is no longer needed in 4.5 following the merge of https://github.com/godotengine/godot/pull/75950. This means modifying the Windows executable metadata (including the icon) now works out of the box with no external setup needed.

However, this issue is still relevant for 3.x, but most feature work has moved on to 4.x at this point.

**Comment by akien-mga** [comment]
rcedit is also no longer needed in 3.7 after #109520.
